db/index.spec.js

```diff
@@ -1,9 +1,9 @@
 const DB = require('./index')
 
 const expectedRow = {
-  code: 'A5G98S4K1',
-  sin: '540739869',
-  dateOfBirth: '1977-09-09',
+  code: 'd75535de98ecea315854491c8d036f8f$ada904924ee513b2d85d4ef775bbd1d835979dd8341047714da8d000ccec8adc',
+  sin: '3c983be4174c2711a44953311ee69792$cd18d36c21efd729f3ee8aab191912fb9467dbaba0552652cc6c451ac8a5031a',
+  dob: '5d3ac41f0d7b0387a651f2ed2b1d883a$fc2a8d33353897b1c6d723904911b91c6edb09283f7784e93ded0ef9dc659ba2',
```

Comment:
`dateOfBirth` here too.

Reply:
hmm so it seems that camelCase [is possible in postgres, but can be a pain](https://stackoverflow.com/questions/11270753/underscores-or-camelcase-in-postgresql-identifiers-when-the-programming-languag). Snake case seems to be more standard, and is what's already happening in our cloud. But, open to changing it!

Reply:
in which case, I'd also change to `first_name`